SettingPresets/SupportedPlatforms/iOS.yml

```diff
@@ -0,0 +1,4 @@
+SUPPORTED_PLATFORMS: iphoneos iphonesimulator
+TARGETED_DEVICE_FAMILY: '1,2'
+SUPPORTS_MACCATALYST: NO
+SUPPORTS_MAC_DESIGNED_FOR_IPHONE_IPAD: YES
```

Comment:
I saw that Xcode 14 does this by default if you create an iOS app, by default it can run on M1+ in a window that follow the iPad design.

If you chose in `supportedPlatforms` that you want to use macOS or Catalysts this will be disable because it is not possible to support multiple Mac platform.